support/hydrationStyles.ts

```diff
@@ -1,33 +0,0 @@
-type Dirent = import("fs").Dirent;
```

Comment:
Maybe we can leave this file for now even though it won't be used until its fixed?